### PR TITLE
custom-element internal classes are switched to i-amphtml- form

### PR DESCRIPTION
**Pull request by dvoytenko**
Partial for #6463.

/cc @honeybadgerdontcare

### CODE REVIEW

File: css/amp.css [hunk 1]
```diff
@@ -148,23 +148,32 @@ i-amphtml-sizer {
...
   left: 0;
   bottom: 0;
   right: 0;
 }
 
-.-amp-replaced-content {
+/* TODO(dvoytenko, #6794): Remove old `-amp-replaced-content` form after the new
+   form is in PROD for 1-2 weeks. */
+.-amp-replaced-content,
+i-amphtml-replaced-content {
```
Comment (honeybadgerdontcare) [inline css/amp.css]: missing `.` as in `.i-amphtml-replaced-content`

File: css/amp.css [hunk 2]
```diff
@@ -148,23 +148,32 @@ i-amphtml-sizer {
   display: block !important;
 }
 
-.-amp-fill-content {
+/* TODO(dvoytenko, #6794): Remove old `-amp-fill-content` form after the new
+   form is in PROD for 1-2 weeks. */
+.-amp-fill-content,
+i-amphtml-fill-content {
```
Comment (honeybadgerdontcare) [inline css/amp.css]: missing `.` as in `.i-amphtml-fill-content`

File: css/amp.css (hunk 1 above)
Comment (dvoytenko) [inline css/amp.css]: Fixed. Thanks! Very good catch!

File: css/amp.css (hunk 2 above)
Comment (dvoytenko) [inline css/amp.css]: Fixed. Thanks! Very good catch!

**Comment by dvoytenko** [comment]
@honeybadgerdontcare PTAL

@jridgewell There will be a major cleanup phase once the changes make to PROD.

**Pull request merged**